web/main/models.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Huh, we need this with 3.9?

Reply:
Yeah, to be able to declare a type that is defined in the same file.